server/README.md

```diff
@@ -0,0 +1,38 @@
+# Hasal Online Server
+
+## Running
+
+By default, the config files will put under `~/.hasal_server` folder.
+
+```bash
+$ pushd server
+$ python server.py <IP>:<PORT>
+  ...
+$ popd
+```
+
+If you want to run it with customized config, please add environment variable `CONFIG_PATH`.
```

Comment:
If you want to run it with customized config, please `set your path to` environment variable `CONFIG_PATH`.